app/assets/javascripts/application.js

```diff
@@ -255,7 +255,7 @@ function template_info(div, url) {
   // Use a post to avoid request URI too large issues with big forms
   $.ajax({
     type: "POST",
-    url: url + "&provisioning=" + build,
+    url: url + ((url.indexOf('?') == -1) ? '?' : '&') + "provisioning=" + build,
```

Comment:
`url.indexOf('?')` should return false if there's a question mark in the `url` string, as that would be interpreted as a query string by the server.  If the path component of the URL was intended to contain a question mark, it would be URL-encoded and not picked up by indexOf.